doc/Preboarding_and_Onboarding.rst

```diff
@@ -5,12 +5,6 @@ Preboarding and Onboarding
 Before executing the **Preboarding** section, it must be ensured that the 28100 port is open. To open the port, run the below mentioned commands.
```

Comment:
Mukul, you removed the lines about how to open the ports.  Do you also need to remove in the previous line where it says, "To open the port, run the below mentioned commands."

Reply:
yes I had removed the previous line too